### ISSUE TITLE
(el-get-bundle elpa:magit) fails to resolve correct dependencies

### ISSUE THREAD

**Issue by d4gg4d** [dimitri/el-get#2859]
### Observed behavior

Noticed that when trying

```elisp
(el-get-bundle magit :type elpa)
```
Resolves to `.status.el` line

```elisp
...
(magit status "installed" recipe
	(:name magit :type elpa :depends
	       (with-editor transient dash))
...
```
Which misses the dependencies of `git-commit` and `magit-section` (see [melpa](https://melpa.org/#/magit)). This causes the magit to fail in the beginning to message of `failure to load magit-section, file not found`.

### Expected behavior

el-get should resolve correct dependencies for the package.

### Workaround

Add following modules manually.

```elisp
(el-get-bundle elpa:magit-section)
(el-get-bundle elpa:git-commit)
```

**Comment by npostavs** [comment]
Yes, currently el-get doesn't understand the dependencies declared for package.el (and it's a bit complicated since packages can be split more finely than their source repositories, as in magit's case).
